backend/src/main/java/com/woowacourse/naepyeon/service/RollingpaperService.java

```diff
@@ -95,6 +99,20 @@ private String findRollingpaperAddresseeNickname(final Rollingpaper rollingpaper
         return teamParticipationRepository.findNicknameByMemberIdAndTeamId(rollingpaper.getAddresseeId(), teamId);
     }
 
+    @Transactional(readOnly = true)
+    public ReceivedRollingpapersResponseDto findReceivedRollingpapers(
+            final Long loginMemberId, final Pageable pageRequest) {
+        final Page<Rollingpaper> rollingpapers = rollingpaperRepository.findByMemberId(loginMemberId, pageRequest);
+        final List<ReceivedRollingpaperResponseDto> receivedRollingpaperResponseDtos = rollingpapers.stream()
+                .map(it -> ReceivedRollingpaperResponseDto.of(it.getId(), it.getTitle(), it.getTeam()))
+                .collect(Collectors.toUnmodifiableList());
+        return new ReceivedRollingpapersResponseDto(
+                rollingpapers.getTotalElements(),
+                rollingpapers.getNumber(),
+                receivedRollingpaperResponseDtos
+        );
```

Comment:
Page번호는 하나 더해서 넘겨주어야 하지 않나요?
```suggestion
        return new ReceivedRollingpapersResponseDto(
                rollingpapers.getTotalElements(),
                rollingpapers.getNumber()+1,
                receivedRollingpaperResponseDtos
        );
```

Reply:
0-index로 되도록 api가 수정됐습니다! 해당 변경사항 반영해서 푸시했습니다!